### PR TITLE
refactor(components)!: sync with design system

### PR DESCRIPTION
**Pull request by moonlitgrace**
https://www.figma.com/design/tNUUtsk20ltOrQICdEoggG/Core-Project?node-id=582-5&t=NAJkvALqjXmUROQN-0

### CODE REVIEW

File: src/components/coreproject-logo-figma/coreproject-logo-figma.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-logo-figma/coreproject-logo-figma.tsx:39]: You know this wont work?

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: src/components/coreproject-shape-bell/coreproject-shape-bell.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-bell/coreproject-shape-bell.tsx:40]: Neither would this

File: src/components/coreproject-shape-book/coreproject-shape-book.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-book/coreproject-shape-book.tsx:40]: Nor this

File: src/components/coreproject-shape-box/coreproject-shape-box.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-box/coreproject-shape-box.tsx:39]: Nor this

File: src/components/coreproject-shape-calendar/coreproject-shape-calendar.tsx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-calendar/coreproject-shape-calendar.tsx:39]: Nor this

File: src/components/coreproject-shape-cast/coreproject-shape-cast.tsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-cast/coreproject-shape-cast.tsx:39]: Nor this

File: src/components/coreproject-shape-check/coreproject-shape-check.tsx [hunk 7]
```diff
@@ -33,11 +33,18 @@ export class CoreprojectShapeMenu {
           ref={el => (this.svg_element = el as SVGElement)}
           width={this?.width}
           height={this?.height}
-          fill="currentColor"
-          viewBox="0 0 48 48"
+          viewBox="0 0 30 30"
+          fill="none"
           xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-check/coreproject-shape-check.tsx:39]: Nor this

File: src/components/coreproject-shape-chevron/coreproject-shape-chevron.tsx [hunk 8]
```diff
@@ -1,38 +1,82 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-chevron/coreproject-shape-chevron.tsx:40]: Nor this

File: src/components/coreproject-shape-chevrons/coreproject-shape-chevrons.tsx [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-chevrons/coreproject-shape-chevrons.tsx:40]: Nor this

File: src/components/coreproject-shape-circle/coreproject-shape-circle.tsx [hunk 10]
```diff
@@ -1,38 +1,52 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-circle/coreproject-shape-circle.tsx:39]: Nor this

File: src/components/coreproject-shape-clipboard/coreproject-shape-clipboard.tsx [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-clipboard/coreproject-shape-clipboard.tsx:39]: Nor this

File: src/components/coreproject-shape-clock/coreproject-shape-clock.tsx [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-clock/coreproject-shape-clock.tsx:39]: Nor this

File: src/components/coreproject-shape-cloud-off/coreproject-shape-cloud-off.tsx [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-cloud-off/coreproject-shape-cloud-off.tsx:39]: Nor this

File: src/components/coreproject-shape-code/coreproject-shape-code.tsx [hunk 14]
```diff
@@ -1,38 +1,59 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-code/coreproject-shape-code.tsx:39]: Nor this

File: src/components/coreproject-shape-copy/coreproject-shape-copy.tsx [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-copy/coreproject-shape-copy.tsx:39]: Nor this

File: src/components/coreproject-shape-corner/coreproject-shape-corner.tsx [hunk 16]
```diff
@@ -0,0 +1,190 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-corner/coreproject-shape-corner.tsx:40]: Nor this

File: src/components/coreproject-shape-delete/coreproject-shape-delete.tsx [hunk 17]
```diff
@@ -1,46 +1,73 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-delete/coreproject-shape-delete.tsx:39]: Nor this

File: src/components/coreproject-shape-download-cloud/coreproject-shape-download-cloud.tsx [hunk 18]
```diff
@@ -0,0 +1,73 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-download-cloud/coreproject-shape-download-cloud.tsx:39]: Nor this

File: src/components/coreproject-shape-download/coreproject-shape-download.tsx [hunk 19]
```diff
@@ -1,46 +1,66 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-download/coreproject-shape-download.tsx:39]: Nor this

File: src/components/coreproject-shape-edit/coreproject-shape-edit.tsx [hunk 20]
```diff
@@ -1,82 +1,93 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-edit/coreproject-shape-edit.tsx:40]: Nor this

File: src/components/coreproject-shape-external-link/coreproject-shape-external-link.tsx [hunk 21]
```diff
@@ -1,46 +1,66 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-external-link/coreproject-shape-external-link.tsx:39]: Nor this

File: src/components/coreproject-shape-eye/coreproject-shape-eye.tsx [hunk 22]
```diff
@@ -0,0 +1,82 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-eye/coreproject-shape-eye.tsx:40]: Nor this

File: src/components/coreproject-shape-fast-forward/coreproject-shape-fast-forward.tsx [hunk 23]
```diff
@@ -0,0 +1,59 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-fast-forward/coreproject-shape-fast-forward.tsx:39]: Nor this

File: src/components/coreproject-shape-file/coreproject-shape-file.tsx [hunk 24]
```diff
@@ -0,0 +1,165 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-file/coreproject-shape-file.tsx:40]: Nor this

File: src/components/coreproject-shape-filter/coreproject-shape-filter.tsx [hunk 25]
```diff
@@ -1,44 +1,52 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-filter/coreproject-shape-filter.tsx:39]: Nor this

File: src/components/coreproject-shape-flag/coreproject-shape-flag.tsx [hunk 26]
```diff
@@ -0,0 +1,60 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this?.width}
+          height={this?.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-flag/coreproject-shape-flag.tsx:39]: Nor this

File: src/components/coreproject-shape-folder/coreproject-shape-folder.tsx [hunk 27]
```diff
@@ -0,0 +1,98 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this.width}
+          height={this.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-folder/coreproject-shape-folder.tsx:40]: Nor this

File: src/components/coreproject-shape-frown/coreproject-shape-frown.tsx [hunk 28]
```diff
@@ -0,0 +1,74 @@
...
+    return (
+      <Host>
+        <svg
+          ref={el => (this.svg_element = el as SVGElement)}
+          width={this.width}
+          height={this.height}
+          viewBox="0 0 30 30"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+          style={{ color: 'currentColor' }}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-frown/coreproject-shape-frown.tsx:39]: Nor this

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

File: src/components/coreproject-logo-figma/coreproject-logo-figma.tsx (hunk 1 above)
Comment (moonlitgrace) [reply src/components/coreproject-logo-figma/coreproject-logo-figma.tsx:39]: Oh yea? i will remove it later

File: src/components/coreproject-shape-link/coreproject-shape-link.tsx [hunk 29]
```diff
@@ -0,0 +1,87 @@
...
+          <defs>
+            <clipPath id="clip0">
+              <rect width="30" height="30" fill="white" />
+            </clipPath>
+          </defs>
+        </svg>
+      </Host>
+    );
+  }
+}
```
Comment (baseplate-admin) [inline src/components/coreproject-shape-link/coreproject-shape-link.tsx:86]: Use if else pattern

File: src/components/coreproject-logo-figma/coreproject-logo-figma.tsx (hunk 1 above)
Comment (baseplate-admin) [reply src/components/coreproject-logo-figma/coreproject-logo-figma.tsx:39]: > Oh yea? i will remove it later

Do same thing twice?

**Pull request closed without merging**